plugins/modules/zos_lineinfile.py

```diff
@@ -790,11 +792,16 @@ def main():
             result['cmd'] = ret['cmd']
         result['changed'] = ret.get('changed', False)
         result['found'] = ret.get('found', 0)
+        result['rc'] = rc
+        result['stdout'] = stdout
+        result['return_content'] = stdout
     # Only return 'rc' if stderr is not empty to not fail the playbook run in a nomatch case
     # That information will be given with 'changed' and 'found'
-    if len(stderr):
-        result['stderr'] = str(stderr)
-        result['rc'] = rc
+    # if len(stderr):
```

Comment:
Why to remove `stderr` assignment to return values? I think this is important to keep